cookbook/absoluteFromLegacyDate.mjs

```diff
@@ -0,0 +1,15 @@
+import { Temporal } from "../polyfill/lib/index.mjs";
```

Comment:
I'd prefer if code samples are written in terms of the new standard (with Temporal already there as a global), rather than importing the polyfill. That is, just deleting this line and leaving everything else the same.

Reply:
After a conversation with Philip and others the decision was to have runnable examples that developers could checkout and run themselves, I think until this is in the spec it needs to import the polyfill for that to happen.

Not sure if @pipobscure has any thoughts

Reply:
Feel free to go ahead with landing this if you prefer.

Reply:
Why not make them runnable by installing a global as part of the runner setup?

Reply:
In a few conversations about this topic, @pipobscure and I agreed that this should be deleted, replaced by a comment explaining how to run this in Node at the command line.

Reply:
> Why not make them runnable by installing a global as part of the runner setup?

Is that possible with something like this https://github.com/tc39/proposal-temporal/blob/main/polyfill/lib/initialise.js which uses a promise, seems like nodejs' preload doesn't wait for any promise to resolve

Reply:
imports have gone and this will be mentioned in the README.md instead